integration_tests/snapshots/logs/process-input-traced_node16.log

```diff
@@ -38,7 +38,6 @@ START
           "_dd.p.tid": "XXXX",
           "_dd.p.dm": "-0",
           "service": "remappedApiGatewayServiceName",
-          "version": "1.0.0",
```

Comment:
Did we figure out why this was removed?

Reply:
Not sure. Looks like it's just removed on the inferred span, but not the aws.lambda spans.

Reply:
Okay, that sounds fine then. I think it'll be important to make sure they remain on the aws.lambda span even if they're removed from the inferred span.